docs/source/getting_started.rst

```diff
@@ -1,12 +1,120 @@
+
+  * :doc:`API basics <api_basics>`
+
+  * :doc:`API interoperability <api_interoperability>`
+
+- `Where to next?`_
+
+  * `Social media`_
+
+  * `Blogs`_
+
+  * `Research`_
+
+  * `Get involved`_
+
+New to vector search?
+=====================
+
+If you are unfamiliar with the basics of vector search or how vector search differs from vector databases, then :doc:`this primer on vector search guide <choosing_and_configuring_index>` should provide some good insight. Another good resource for the uninitiated is our :doc:`vector databases vs vector search <vector_databases_vs_vector_search>` guide. As outlined in the primer, vector search as used in vector databases is often closer to machine learning than to traditional databases. This means that while traditional databases can often be slow without any performance tuning, they will usually still yield the correct results. Unfortunately, vector search indexes, like other machine learning models, can yield garbage results of not tuned correctly.
+
+Fortunately, this opens up the whole world of hyperparamer optimization to improve vector search performance and quality. Please see our :doc:`index tuning guide <tuning_guide>` for more information.
+
+When comparing the performance of vector search indexes, it is important that considerations are made with respect to three main dimensions:
+
+#. Build time
+#. Search quality
+#. Search performance
+
+Please see the :doc:`primer on comparing vector search index performance <comparing_indexes>` for more information on methodologies and how to make a fair apples-to-apples comparison during your evaluations.
+
+Supported indexes
+=================
+
+cuVS supports many of the standard index types with the list continuing to grow and stay current with the state-of-the-art. Please refer to our :doc:`vector search index guide <indexes/indexes>` for to learn more about each individual index type, when they can be useful on the GPU, the tuning knobs they offer to trade off performance and quality.
+
+The primary goal of cuVS is to enable speed, scale, and flexibility (in that order)- and one of the important value propositions is to enhance existing software deployments with extensible GPU capabilities to improve pain points while not interrupting parts of the system that work well today with CPU.
+
+
+Using cuVS APIs
 ===============
 
-This guide provides an initial starting point of the basic concepts and using the various APIs in the cuVS software development kit.
+cuVS is a C++ library its core, which is wrapped with a C library and exposed further through various different languages. cuVS currently provides APIs and documentation for :doc:`C <c_api>`, :doc:`C++ <cpp_api>`, :doc:`Python <python_api>`, and :doc:`Rust <rust_api/index>` with more languages in the works. our :doc:`API basics <api_basics>` provides some background and context about the important paradigms and vocabulary types you'll encounter when working with cuVS types.
```

Comment:
`cuVS is a C++ library its core` doesn't sound quite right to me - maybe should be `cuVS is a C++ library at its core` instead?